src/panels/config/integrations/integration-panels/zwave_js/zwave_js-node-config.ts

```diff
@@ -345,7 +345,10 @@ class ZWaveJSNodeConfig extends SubscribeMixin(LitElement) {
     if (ev.target === undefined || this._config![ev.target.key] === undefined) {
       return;
     }
-    if (this._config![ev.target.key].value === ev.target.value) {
+    if (
+      this._config![ev.target.key].value?.toString() ===
+      ev.target.value.toString()
```

Comment:
isn't it already a string?
```suggestion
      ev.target.value
```

Reply:
Good point.